.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         override: true
     - name: Build ${{ matrix.profile }}
       run: |
-        cargo build --profile=${{ matrix.profile }} --features=dont-generate-test-files --features=${{ matrix.features }} --lib --tests --examples
+        cargo build --profile=${{ matrix.profile }} --features=${{ matrix.features }} --lib
```

Comment:
will we build-test examples in some other action?

Reply:
Yeah, the `clippy` step does basically do that. The reason I removed those two here is because they'd trigger a dev build, actually. For this build step, I think there is value in building what consumers of the crate would also build. But we could conceivably introduce a separate step as well.